.ci/templates/common-validation-stages.yml

```diff
@@ -0,0 +1,51 @@
+parameters:
+- name: moduleFolderName
+  type: string
+- name: moduleDisplayName
+  type: string
+
+stages:
+- stage: 'Validation_${{ parameters.moduleDisplayName }}' # Valid stage name cannot contain the character '.'
+  displayName: 'Validate Package ${{ parameters.moduleFolderName }}'
+  pool:
+    vmImage: ubuntu-20.04
+  jobs:
+  - job: Validate
+    displayName: 'Validate ${{ parameters.moduleFolderName }}'
+    steps:
+    - task: AzureCLI@2
+      inputs:
+        azureSubscription: Scripting Metadata
+        scriptLocation: 'inlineScript'
+        scriptType: 'pscore'
+        inlineScript: |
+          $accessToken = az account get-access-token --query accessToken --resource 499b84ac-1321-427f-aa17-267ca6975798 -o tsv
```

Comment:
Would it be better to store the Resource GUID value in vars.yml instead? May not be required though if this value will not change in the future